packages/react-charting/src/components/Legends/Legends.types.ts

```diff
@@ -220,10 +215,28 @@ export interface ILegendsProps {
 
   /**
    * Defines the function that is executed upon hiding of hover card
-   * make sure to send prop  when  the prop  is canSelectMultipleLegends is set to ture
-   * and empty the selecetd state legends
+   * make sure to send prop  when  the prop  is canSelectMultipleLegends is set to true
+   * and empty the selected state legends
    */
   onLegendHoverCardLeave?: VoidFunction;
+
+  /**
+   * Callback issued when the selected option changes.
+   */
+  onChange?: (selectedLegends: string[], event: React.MouseEvent<HTMLButtonElement>, currentLegend?: ILegend) => void;
```

Comment:
Can be renamed to something like `onLegendSelectionChange` for better clarity

Reply:
`onChange` is a common naming for input controllers.